### PR TITLE
py sol for APSP

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

AC: https://open.kattis.com/submissions/14638901

### CODE REVIEW

File: content/6_Advanced/SPneg.mdx [hunk 1]
```diff
@@ -393,7 +393,64 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+INF = 10**18
+MAX_N = 150
```
Comment (SansPapyrus683) [inline content/6_Advanced/SPneg.mdx:400]: is max_n even used

File: content/6_Advanced/SPneg.mdx [hunk 2]
```diff
@@ -393,7 +393,64 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+INF = 10**18
```
Comment (SansPapyrus683) [inline content/6_Advanced/SPneg.mdx:399]: float("inf")

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: content/6_Advanced/SPneg.mdx (hunk 1 above)
Comment (freakin23) [reply content/6_Advanced/SPneg.mdx]: okay removed.

File: content/6_Advanced/SPneg.mdx (hunk 2 above)
Comment (freakin23) [reply content/6_Advanced/SPneg.mdx:399]: the thing is that with float('inf') it gives TLE.

TLE with float('inf'): https://open.kattis.com/submissions/14642683

AC (without float('inf'): https://open.kattis.com/submissions/14642731

Comment (SansPapyrus683) [reply content/6_Advanced/SPneg.mdx:399]: can you mention that using float(inf) gives tle

Comment (freakin23) [reply content/6_Advanced/SPneg.mdx:399]: Like adding a note to it.

Comment (SansPapyrus683) [reply content/6_Advanced/SPneg.mdx:399]: a comment

Comment (freakin23) [reply content/6_Advanced/SPneg.mdx]: done

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**